slc7-builder/Dockerfile.arm64

```diff
@@ -0,0 +1,41 @@
+FROM centos/aarch64
+ADD cernca.repo /etc/yum.repos.d/cernca.repo
+ADD  GPG-KEY-EUGridPMA-RPM-3 /etc/pki/rpm-gpg/GPG-KEY-EUGridPMA-RPM-3
+RUN rpm -iUv http://dl.fedoraproject.org/pub/epel/7/aarch64/e/epel-release-7-9.noarch.rpm
+RUN rpm -iUvh http://linuxsoft.cern.ch/cern/slc65/updates/i386/RPMS/CERN-CA-certs-20160421-2.slc6.noarch.rpm
+RUN yum update -y && yum clean all
```

Comment:
You can grab it from here: `http://davidlt.web.cern.ch/davidlt/vault/dumb-init/v1.2.0/dumb-init`